<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserCenter</class>
 <widget class="QWidget" name="UserCenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>个人中心</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 173);</string>
  </property>
  <widget class="QLabel" name="avatar_lbl">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>80</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="changeUname_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
    </font>
   </property>
   <property name="text">
    <string>修改用户名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changePwd_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
    </font>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcome_lbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
